plancton/__init__.py

```diff
@@ -210,18 +211,19 @@ def _read_conf(self):
       self.conf["max_docks"] = 0
     if not isinstance(self.conf["docker_cmd"], list):
       self.conf["docker_cmd"] = self.conf["docker_cmd"].split(" ")
-    if self.conf["influxdb_url"] and not "#" in self.conf["influxdb_url"]:
-      # Invalid InfluxDB URL: disable monitoring
-      self.conf["influxdb_url"] = None
-    self.logctl.debug("Configuration:\n%s" % json.dumps(self.conf, indent=2))
+    tmp_url_list = []
+    if isinstance(self.conf["influxdb_url"], str):
+      self.conf["influxdb_url"] = set(self.conf["influxdb_url"])
+    if isinstance(self.conf["influxdb_url"], set):
```

Comment:
And, this is why tests are useful! So I would really like to see a Travis test vouching for correctness in this case.